src/main/app/src/app/informatie-objecten/informatie-object-view/informatie-object-view.component.ts

```diff
@@ -135,7 +135,7 @@ export class InformatieObjectViewComponent  extends ActionsViewComponent impleme
     }
 
     private toevoegenNieuweVersieActie() {
-        if (this.laatsteVersieInfoObject.status !== InformatieobjectStatus.DEFINITIEF) {
+        if (this.laatsteVersieInfoObject.status !== InformatieobjectStatus.DEFINITIEF && this.zaken?.length > 0) {
```

Comment:
Zet er even een regel commentaar bij dat het gaat om inbox of ontkoppelde documenten. het is niet duidelijk dat het om inbox of ontkoppelde documenten gaat als je this.zaken?.length > 0 leest

Reply:
Toegevoegd